.github/workflows/scripts/update-event-bridge-target.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
I am not fully getting what this one does. Can I bother you with some clarification please?

Reply:
I added some comments around this, but the schedule task trigger in EventBridge points at a specific ECS task definition. That task definition is where the tag for the ECR image is defined.

We update the task-definition in the core , but since there is no service for these tasks deploying a change to the task definition won't actually update the target of the job. This script updates that target ARN for all scheduled jobs.

Without this, then all jobs would run in an ECR image with the tag `live`. Which probably wouldn't hurt anything, but isn't as explicit as using the git SHA based image tag.

Does that (and the comments) provide clarity?

Reply:
[This is called](https://github.com/techmatters/hrm/blob/bo/feat/CHI-1778-transcript-cleanup/.github/workflows/hrm-ecs-deploy.yml#L185) as part of the hrm-ecs-deploy workflow.